api/src/main/java/au/org/aodn/nrmn/restapi/service/validation/SiteValidation.java

```diff
@@ -92,6 +92,13 @@ public SurveyValidationError validateSurveyAtSite(StagedRowFormatted row) {
                 return new SurveyValidationError(ValidationCategory.DATA, ValidationLevel.WARNING, message,
                         Arrays.asList(row.getId()), Arrays.asList("latitude", "longitude"));
             }
+            // If survey is less than 10 meters, warn users survey lat & lon will be removed.
```

Comment:
@HavierD I don't think this warning is necessary. However, there should be a warning when survey coordinates  are outside of a 10m radius  from site coords , and the warning should display the difference in metres

Reply:
@bpasquer Got it. Will change it shortly.